tests/test_rss.py

```diff
@@ -0,0 +1,1291 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from sopel.db import SopelDB
+from MirahezeBot-Plugins import sopel.plugins-rss
```

Comment:
travis says invalid syntax on this line

Reply:
I know. I'm trying to work out the correct one.